server/src/helpers/jwt.ts

```diff
@@ -0,0 +1,16 @@
+import Jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import { Payload } from '../interfaces';
+
+dotenv.config();
```

Comment:
```suggestion
```
No need to import it every where as import it at the app level will expose it where we need